compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -1904,9 +1904,7 @@ impl<'tcx> TyCtxt<'tcx> {
     ) -> (Ident, DefId) {
         let scope =
             match ident.span.normalize_to_macros_2_0_and_adjust(self.expn_that_defined(scope)) {
-                Some(actual_expansion) => {
-                    self.hir().definitions().parent_module_of_macro_def(actual_expansion)
-                }
+                Some(actual_expansion) => actual_expansion.expn_data().parent_module.unwrap(),
                 None => self.parent_module(block).to_def_id(),
             };
```

Comment:
Encountering an expansion without a parent module (typically an AST pass), should be ok here.
```rust
        let scope = ident
            .span
            .normalize_to_macros_2_0_and_adjust(self.expn_that_defined(scope))
            .and_then(|expn_id| expn_id.expn_data().parent_module)
            .unwrap_or_else(|| self.parent_module(block).to_def_id());
```
See the branch https://github.com/petrochenkov/rust/commit/89f28456875e85487109e995d25af79db9187728